terraform/modules/region/sns.tf

```
@@ -0,0 +1,3 @@
+data "aws_sns_topic" "cloudwatch_api" {
+  name = "CloudWatch-API-to-PagerDuty-local-${data.aws_region.current.name}"
```

This needs to use environment_name again so that it works in real infra (where environment_name will be e.g. "production")
```suggestion
  name = "CloudWatch-API-to-PagerDuty-${local.environment_name}-${data.aws_region.current.name}"
```